chain/network/src/peer_manager.rs

```diff
@@ -434,11 +547,12 @@ impl PeerManagerActor {
         // If the last edge we have with this peer represent a connection addition, create the edge
         // update that represents the connection removal.
         self.active_peers.remove(&peer_id);
+        self.routing_table.remove_peer(&peer_id);
```

Comment:
Is this some existing bug?

Reply:
No there was no bug, in this change I added storing IBF for each peer. I added `remove_peer` method for removing stored `ibf` data attached to peer.